### ISSUE TITLE
User-facing documentation

### ISSUE THREAD

**Issue by tomrijnbeek** [beardgame/utilities#76]
I propose having some user-facing documentation using GitHub pages. I prefer the solution where the Markdown for this documentation is put into the /doc folder of the repository, where the documentation can be updated in the same PR as the functionality is added/edited.

**Comment by amulware** [comment]
I agree with having the documentation be in the repository, and in markdown.

**Comment by tomrijnbeek** [comment]
I will set up _a thing_ then. I am not entirely sure what exactly we need to put in the docs for now, but I will populate it with a high-level overview of the library.
